### ISSUE TITLE
Request for new release

### ISSUE THREAD

**Issue by UebelAndre** [oli-obk/cargo_metadata#208]
Would it be possible to get a `0.16` or something? I'm hoping to use https://github.com/oli-obk/cargo_metadata/commit/0d034812718cf11fd3dc1cfac8b9612566634726 which does not appear to be in `0.15`

**Comment by oli-obk** [comment]
I need to set up cargo semver. I keep not knowing whether I need a breaking bump or not. Ping me if nothing happens by next week

**Comment by msrd0** [comment]
The tag for 0.15 is missing, but according to [docs.rs](https://docs.rs/crate/cargo_metadata/latest/source/.cargo_vcs_info.json) it was c800efe4d658ba43b8d43385acbd55dcbd84636d. Looking through the [diff](https://github.com/oli-obk/cargo_metadata/compare/c800efe4d658ba43b8d43385acbd55dcbd84636d..main), it looks like there were no breaking changes made, eventhough the changelog mentions removed reexports.

**Comment by oli-obk** [comment]
```
> cargo semver-checks check-release
    Updating index
     Parsing cargo_metadata v0.15.1 (current)
     Parsing cargo_metadata v0.15.0 (baseline)
    Checking cargo_metadata v0.15.0 -> v0.15.1 (minor change)
   Completed [   0.211s] 20 checks; 20 passed, 0 skipped
```

Published 0.15.1

**Comment by UebelAndre** [comment]
Thank you! The release of `0.15.1` looks good on my end. Closing this issue out 😄
